### PR TITLE
{compiler}[GCCcore/10.2.0] Clang v11.0.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3551.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/ab4262258fd7ee61b4ae6299ab6264f4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node2614.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/d6f522a204f6beee45c9429a4f8feba7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3100.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/b838e926837a2bb520952e4d977f79b3 for a full test report.

File: easybuild/easyconfigs/c/Clang/Clang-11.0.1-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    ('CMake', '3.18.4'),
+    ('Python', '3.8.6'),
+]
+
+assertions = True
+usepolly = True
+build_lld = True
+libcxx = True
+enable_rtti = True
+build_extra_clang_tools = True
```
Comment (verdurin) [inline easybuild/easyconfigs/c/Clang/Clang-11.0.1-GCCcore-10.2.0.eb:67]: @boegel does this rely on a new easyblock? I saw this warning when kicking off a test build using `4.3.2`:
```
WARNING: Use of 1 unknown easyconfig parameters detected in Clang-11.0.1-GCCcore-10.2.0.eb: build_extra_clang_tools
```

Comment (boegel) [reply easybuild/easyconfigs/c/Clang/Clang-11.0.1-GCCcore-10.2.0.eb:67]: Yes, this requires https://github.com/easybuilders/easybuild-easyblocks/pull/2310, please try again with EasyBuild v4.3.3?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/11404eb821d5f1edef8a47571bfcf74f for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
nuc.lan - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.7
See https://gist.github.com/1c1a4b5bcd29c0f031d966346224ccda for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**